servicecontrol/plugins.md

```diff
@@ -106,43 +89,69 @@ As mentioned above, there are two types of custom checks:
 * Custom check that runs once, when the endpoint host starts
 * Periodic check that runs at defined intervals
  
-The result of a custom check is either success or a failure (with a detailed description defined by the developer). This result is sent as a message to the ServiceControl queue.   
+The result of a custom check is either success or a failure (with a detailed description defined by the developer). This result is sent as a message to the ServiceControl queue and status will be shown in the ServicePulse UI.   
 
 **Related articles**
 
 - [How to Develop Custom Checks for ServicePulse](/servicepulse/how-to-develop-custom-checks.md)
 
-### ServiceControl SagaAudit Plugin
+NOTE: It is essential that you deploy this plugin to your endpoint in production in order to receive error notifications about the custom check failures in the ServicePulse dashboard.
 
-The Saga Audit plugin collects saga runtime activity information. This information enables the display of detailed saga data, behavior, and current status in the ServiceInsight Saga View. The plugin sends the relevant saga state information as messages to the ServiceControl queue whenever a saga state changes. This enables the Saga View to be highly detailed and up-to-date.
+#### Relevant Nuget packages for the CustomChecks Plugin
 
-However, depending on the saga's update frequency, it may also result in a large number of messages, and a higher load on both the sending endpoint and on the receiving ServiceControl instance. As a result, prior to deploying the Saga Audit plugin, you should test to verify that the Saga Audit plugin communication overhead does not interfere with your expected endpoint performance.   
+| **Endpoint NSB Version** | **NuGet** | 
+|:----- |:----- |
+|NSB v5.x|[ServiceControl.Plugin.Nsb5.CustomChecks](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb5.CustomChecks)|
+|NSB v4.x|[ServiceControl.Plugin.Nsb4.CustomChecks](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb4.CustomChecks)|
+|NSB v3.x|[ServiceControl.Plugin.Nsb3.CustomChecks](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb3.CustomChecks)|
+
+#### Deprecated Nuget
+If you are using the older version of the plugin, namely **ServiceControl.Plugin.CustomChecks** please remove the package and replace it with the appropriate plugin based on your NServiceBus version. This package has been deprecated and unlisted.
+
+### SagaAudit Plugin for Saga Visualization in ServiceInsight
```

Comment:
Maybe rename to "SagaAudit Plugin for visualizing sagas in ServiceInsight (For Development ONLY)"